addons/hr_expense/models/hr_employee.py

```diff
@@ -3,6 +3,33 @@
 from odoo import fields, models, api
 
 
+class EmployeeBase(models.AbstractModel):
+    _inherit = 'hr.employee.base'
+
+    filter_for_expense = fields.Boolean(store=False, search='_search_filter_for_expense')
+
+    def _search_filter_for_expense(self, operator, value):
+        assert operator in ('=', '!=') and value, "Operation not supported"
```

Comment:
Am I missing something? You never use the values for `operator` or `value`, how can this work?

Reply:
The point of this search is to implement the domain here: https://github.com/odoo/odoo/blob/6faff899a9c662e12b3db67cd3f0ed813aa9dfe3/addons/hr_expense/models/hr_expense.py#L41
Which is why this worked.

And since the field is Boolean, asserting `value`, is equivalent to constraining it to `True`.

But now that I think of it, since as of yesterday we accept `!=`, at least conceptually, we do have to invert the result based on the operator.

So here to me the solution is either:
* forbid `!=` as before
* Also allow `value = False` and inverse the returned domain based on a combination of the 2.

Reply:
I re-forbade `!=`